src/LoginPage/LoginPage.jsx

```diff
@@ -1,20 +1,55 @@
 import styled from '@emotion/styled';
 
+import { useRef, useEffect } from 'react';
+
 import LoginFormContainer from './LoginFormContainer';
 
+const DeleteAll = styled.div({
+  position: 'fixed',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: '100%',
+  height: '100%',
+  backdropFilter: 'blur(0.8px)',
+});
+
 const Container = styled.div({
+  position: 'fixed',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
   width: '418px',
   height: '420px',
-  backgroundColor: '#000000',
+  margin: '0',
   color: '#EAEAEA',
+  backgroundColor: '#000000',
 });
 
-export default function LoginPage() {
+export default function LoginPage({
+  onClick,
+}) {
+  const refLogin = useRef();
+
+  function listener({ target }) {
+    if (refLogin.current.contains(target)) { return; }
```

Comment:
```suggestion
    if (refLogin.current.contains(target)) return; 
```

한줄에 다 작성할 때는 이렇게 작성해주셔도 될겁니당!!ㅎㅎ

Reply:
if 문의 가독성을 위해서 중괄호를 생략하지 않고 항상 사용했는데 한 줄로 할 때는 생략하는 편이 좋을 것 같군용 ㅎㅎ

Reply:
그런데 다희님이 그렇다면 그런거라.. ㅎ